File: 0075-sort-colors/0075-sort-colors.py
class Solution(object):
    def sortColors(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        
        start = 0
        
        while start < len(nums)-1:
            for n in range(0, len(nums)-start - 1):
                if nums[n] > nums[n + 1]:
                    nums[n],nums[n+1] = nums[n+1], nums[n]
            start += 1

File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int current = nums.length -1;
        
        while(current > 0){
            int index = current;
            for(int i = current; i >= 0; i--){
                if(nums[i] > nums[index]){
                    index = i;
                }
            }
            if(nums[current] < nums[index]){
                int hold = nums[current];
                nums[current] = nums[index];
                nums[index] = hold;
            }
            current--;
        }
    }
}